src/Components/Users/UserProfile.tsx

```diff
@@ -249,10 +249,12 @@ export default function UserProfile() {
             errors[field] = "Field is required";
             invalidForm = true;
           } else if (
-            states.form.user_type === "Doctor" &&
-            Number(states.form.doctor_experience_commenced_on) > 100
+            (states.form.user_type === "Doctor" &&
+              Number(states.form.doctor_experience_commenced_on) > 100) ||
```

Comment:
> "Doctor experience should be at least 0 years and less than 100 years."

According to the error, shouldn't the condition here be `>= 100` ? (If equal or more than 100, tell them that it needs to be LESS than 100)

```suggestion
              Number(states.form.doctor_experience_commenced_on) >= 100) ||
```

Reply:
Yes, that can be added. However it was already there like this only, so i only added the < 0 condition and didn't touched this condition.

Sure, let me change that !

Reply:
Done @Ashesh3